demisto_sdk/commands/prepare_content/preparers/marketplace_incident_to_alert_playbooks_prepare.py

```diff
@@ -12,7 +13,8 @@ class MarketplaceIncidentToAlertPlaybooksPreparer:
     @staticmethod
     def prepare(
         data: dict,
-        marketplace: MarketplaceVersions = MarketplaceVersions.XSOAR,
+        current_marketplace: MarketplaceVersions = MarketplaceVersions.XSOAR,
+        supported_marketplaces=None,
```

Comment:
please add a type hint

Reply:
This should be a list with the default value of `list(MarketplaceVersions)`, but setting this here will raise a `Default argument value is mutable `. Therefore, I set it here with None and place the default values in the first line in the function. Do you know a better way?